### DISCUSSION TITLE
Data synchronization

### DISCUSSION THREAD

**Discussion by XTUxiongda** [minio/minio#15703] [Q&A] [answered]
I want to synchronize other stored data to mini incrementally in real time. Any suggestions?
At the same time, if I want to achieve synchronization between mini clusters, do you have any suggestions?
For example, the minio cluster in the host room and the minio cluster in the disaster recovery room are directly synchronized

**Comment by harshavardhana** [comment, marked as answer]
You may be looking for server side replication https://docs.min.io/minio/baremetal/replication/replication-overview.html#minio-replication-overview
